CS483/CS483/Kartaclysm/Components/ComponentKartController.cpp

```diff
@@ -457,16 +459,18 @@ namespace Kartaclysm
 	{
 		m_pGameObject->GetTransform().Translate(m_pOutsideForce * p_fDelta);
 		m_pOutsideForce *= powf(2.718281828f, p_fDelta * m_fOutsideForceAccelerationStat);
+
+		float m_fSpin = m_fSpinout * m_fSpinFactor;
 		
 		m_pGameObject->GetTransform().TranslateXYZ(m_fSpeed * sinf(m_fDirection) * p_fDelta, p_fHeightMod, m_fSpeed * cosf(m_fDirection) * p_fDelta);
 		// swerve temporarily disabled until the camera transform heirarchy is fixed
 		if (m_bWheelie)
 		{
-			m_pGameObject->GetTransform().SetRotation(glm::quat(glm::vec3(PI * -0.15f, m_fDirection + m_fSwerve, 0.0f)));
+			m_pGameObject->GetTransform().SetRotation(glm::quat(glm::vec3(PI * -0.15f, m_fDirection + m_fSwerve + m_fSpin, 0.0f)));
```

Comment:
Why not replace m_fSwerve + m_fSpin with a call to GetRotationMod()? It would simplify the process if we needed to add other rotation mods later.